src/components/@shared/AvailableNetworks/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement } from 'react'
+import styles from './index.module.css'
+import NetworkOptions from './networkOptions'
```

Comment:
consistency, `networkOptions.tsx` should be named `NetworkOptions.tsx` file name needs to be capitalized. Be aware this is not that simple to do on case-insensitive systems so most likely you have to use [`git mv`](https://www.hanselman.com/blog/git-is-casesensitive-and-your-filesystem-may-not-be-weird-folder-merging-on-windows)